ext/certstore/loader.c

```diff
@@ -242,6 +242,8 @@ rb_win_certstore_loader_find_certificate(VALUE self, VALUE rb_thumbprint)
               pContext);
 
   if (!pContext) {
+    ALLOCV_END(vThumbprint);
```

Comment:
Probably it's not good because the `winThumbprint` which is the content of `vThumbprint` is still used in following `rb_raise()`

Reply:
Hmm..., I see.